addons/mail/models/mail_scheduled_message.py

```diff
@@ -153,6 +163,8 @@ def write(self, vals):
         res = super().write(vals)
         if new_scheduled_date := vals.get('scheduled_date'):
             self.env.ref('mail.ir_cron_post_scheduled_message')._trigger(fields.Datetime.to_datetime(new_scheduled_date))
+        for scheduled_message in self:
+            scheduled_message._bus_send_store(self)
```

Comment:
I think the intent was to send the scheduled_message, not self?
```suggestion
        for scheduled_message in self:
            scheduled_message._bus_send_store(scheduled_message)
```